packages/utils/src/address.ts

```diff
@@ -11,3 +11,5 @@ export const privateKeyToAddress = (privateKey: string) => {
     '0x' + privateToAddress(Buffer.from(privateKey.slice(2), 'hex')).toString('hex')
   )
 }
+
+export { toChecksumAddress } from 'ethereumjs-util'
```

Comment:
i don't see ethereumsjs-utils in utils pkg.json...

Reply:
https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/package.json#L25 ?

Reply:
sorry i was looking at the wrong package.json